Mathlib/Data/List/Basic.lean

```diff
@@ -972,6 +955,7 @@ theorem indexOf_inj [DecidableEq α] {l : List α} {x y : α} (hx : x ∈ l) (hy
       simp only [h]
     simp only [indexOf_get] at x_eq_y; exact x_eq_y, fun h => by subst h; rfl⟩
 
+@[deprecated getElem_reverse_aux₂ (since := "2024-08-15")]
 theorem getElem_reverse_aux₂ :
```

Comment:
Is this deprecated in favor of the same lemma?

Reply:
Oops, should just be deprecated without a suggestion: this is just an auxiliary lemma for a now deprecated theorem, `get_reverse`.